iothub/service/src/IotHubServiceClient.cs

```diff
@@ -147,6 +153,7 @@ private void InitializeSubclients()
         {
             Devices = new DevicesClient(_hostName, _credentialProvider, _httpClient, _httpRequestMessageFactory);
             Modules = new ModulesClient(_hostName, _credentialProvider, _httpClient, _httpRequestMessageFactory);
+            Configurations = new ConfigurationsClient(_credentialProvider, new HttpTransportSettings());
```

Comment:
You should be passing in the exact same parameters to ConfigurationsClient as are being passed into DevicesClient and ModulesClient here

Reply:
added the relevant changes